spec/linter-tslint-spec.js

```diff
@@ -39,4 +39,20 @@ describe('The TSLint provider for Linter', () => {
       }),
     );
   });
+
+  it('handles null filepath', () => {
+    waitsForPromise(() =>
+      atom.workspace.open(null).then(editor => lint(editor)).then((result) => {
```

Comment:
I feared that might be the case. Ideally the editor input would be mocked out so we can simulate the exact state. I will remove the dupe test =]